pkg/firestore/util.go

```diff
@@ -0,0 +1,51 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package firestore
+
+import (
+	"context"
+	"fmt"
+
+	"cloud.google.com/go/firestore"
+)
+
+// [START firestore_data_custom_type_definition]
+
+const (
+	Collection             = "JVS"
+	JustificationConfigDoc = "JustificationConfig"
+	PublicKeyConfigDoc     = "PublicKeyConfig"
+	CertRotationConfigDoc  = "CertRotationConfig"
+)
+
+type KMSConfig struct {
```

Comment:
I think this type is orthogonal to firestore. Thus we should define it in `pkg/config` package.